jobs/sql/extractions/users.sql

```diff
@@ -3,7 +3,8 @@ SET sql_safe_updates = 0;
 DROP TEMPORARY TABLE IF EXISTS temp_users;
 
 CREATE TEMPORARY TABLE temp_users (
-    username            varchar(50) primary key,
+    user_id             int,
+    username            varchar(50),
```

Comment:
This has the side effect of also fixing the non-unique username issue we are seeing on some systems.